crud-cert-auth/src/main/webapp/WEB-INF/web.xml

```diff
@@ -76,5 +71,19 @@ xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns
     <param-name>resteasy.providers</param-name>
     <param-value>com.restcompress.provider.LZFEncodingInterceptor,com.restcompress.provider.LZFDecodingInterceptor</param-value>
   </context-param>
+  
+  <listener>
+    <listener-class>
+      com.redhat.lightblue.rest.crud.cert.auth.health.CrudCertAuthHealthCheckServletContextListener
+    </listener-class>
+  </listener>
+  <servlet>
+    <servlet-name>healthCheck</servlet-name>
+    <servlet-class>com.codahale.metrics.servlets.HealthCheckServlet</servlet-class>
+  </servlet>
+  <servlet-mapping>
+    <servlet-name>healthCheck</servlet-name>
+    <url-pattern>/healthcheck</url-pattern>
```

Comment:
I think this new health check should be reachable on /health (without any v2 or complete adornments), and we should modify existing health.jsp to be version.jsp and serve it up at /version, as knowing the version deployed is still useful.  Also if we keep it at /health, we won't have to change our existing load balancer configurations to hist the new health check, they will start doing it as soon as we deploy this version  :-)